### PR TITLE
Location context

### PR DESCRIPTION
**Pull request by maxaleks**

### CODE REVIEW

File: src/bot/Bot.js [hunk 1]
```diff
@@ -127,6 +127,12 @@ class Bot {
           ],
           this._controllers.onboarding
         )
+        .when(
+          [
+            new QuickReplyPayloadPatternCommand('SEARCH_LOOKING_FOR_', 'commonHandle')
```
Comment (MaksimAbramchuk) [inline src/bot/Bot.js]: what does `commonHandle` mean? it's not straightforward

File: src/bot/controllers/SearchController.js [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
             $.appUserId,
             this.i18n('search.whatAreYouLooking', [$.sender.username]),
             [
-              this.base.reply.postback('Breakfast 🍳', 'SEARCH_LOOKING_FOR_BREAKFAST'),
-              this.base.reply.postback('Lunch 🌭', 'SEARCH_LOOKING_FOR_LUNCH'),
-              this.base.reply.postback('Dinner 🍽', 'SEARCH_LOOKING_FOR_DINNER'),
-              this.base.reply.postback('Coffee & Tea ☕️', 'SEARCH_LOOKING_FOR_COFFE_AND_TEA'),
-              this.base.reply.postback('Nightlife 🍹', 'SEARCH_LOOKING_FOR_NIGHTLIFE'),
-              this.base.reply.postback('Things to do 🍿', 'SEARCH_LOOKING_FOR_THINGS_TO_DO'),
+                this.base.reply.postback('Breakfast 🍳', 'SEARCH_LOOKING_FOR_BREAKFAST'),
```
Comment (MaksimAbramchuk) [inline src/bot/controllers/SearchController.js]: all the button names are usually stored in i18n files too

File: src/bot/controllers/SearchController.js [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
+                this.base.reply.postback('Lunch 🌭', 'SEARCH_LOOKING_FOR_LUNCH'),
+                this.base.reply.postback('Dinner 🍽', 'SEARCH_LOOKING_FOR_DINNER'),
+                this.base.reply.postback('Coffee & Tea ☕️', 'SEARCH_LOOKING_FOR_COFFE_AND_TEA'),
```
Comment (MaksimAbramchuk) [inline src/bot/controllers/SearchController.js]: `COFFEE`

File: src/integrations/smooch/SmoochApi.js [hunk 4]
```diff
@@ -29,7 +29,30 @@ class SmoochApi {
             });
     }
 
-    sendLocationRequest(userId, coordinates) {
+    sendLocationRequest(userId, text, locationText) {
+        return this.showTypingIndicator(userId)
+            .then(() => {
```
Comment (MaksimAbramchuk) [inline src/integrations/smooch/SmoochApi.js]: Are you sure that indentation is right here?

File: src/locales/en.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
 {
   "randomInput": "I don't understand",
+  "sendLocation": "Send Location",
```
Comment (MaximAbramchuck) [inline src/locales/en.json]: why is it stored separately? maybe move it to the `onboarding` object since it's used inside `OnboardingController`?

**Review by MaksimAbramchuk — CHANGES_REQUESTED** [review]

**Review by MaksimAbramchuk — APPROVED** [review]

**Pull request merged**